Adding CSCW 2023 Paper

katelyn98 opened the pull request:
Adding paper pdf and figures for upcoming paper at CSCW 2023

domoritz commented:
The thumbnail image is a bit large in file size. Can you make it smaller?

domoritz commented:
Especially the thumbnail needs to be smaller since it appears on a page with lots of images. We do run some automatic compression on it later so it will get smaller. For reference, here are some of the existing thumbnails. The thumbnail really should be a recognizable illustration so it shouldn't be too detailed. 

<img width="614" alt="Screenshot 2022-12-20 at 22 28 10" src="https://user-images.githubusercontent.com/589034/208813906-fd2b78ce-f7b3-4abe-a37c-3d1fb26df5b8.png">

katelyn98 commented:
> Especially the thumbnail needs to be smaller since it appears on a page with lots of images. We do run some automatic compression on it later so it will get smaller. For reference, here are some of the existing thumbnails. The thumbnail really should be a recognizable illustration so it shouldn't be too detailed.
> 
> <img alt="Screenshot 2022-12-20 at 22 28 10" width="614" src="https://user-images.githubusercontent.com/589034/208813906-fd2b78ce-f7b3-4abe-a37c-3d1fb26df5b8.png">

Ahhh okay thanks! I changed the image and adjusted the size. Let me know if this works better. Thanks!
